diff -U3 -r zeromq-4.3.4-orig/src/clock.cpp zeromq-4.3.4/src/clock.cpp
--- zeromq-4.3.4-orig/src/clock.cpp	2021-01-03 21:46:01.000000000 +0000
+++ zeromq-4.3.4/src/clock.cpp	2021-07-01 20:27:35.348783385 +0100
@@ -128,7 +128,9 @@
 
 const uint64_t usecs_per_msec = 1000;
 const uint64_t usecs_per_sec = 1000000;
+#ifndef ZMQ_HAVE_OSX
 const uint64_t nsecs_per_usec = 1000;
+#endif
 
 zmq::clock_t::clock_t () :
     _last_tsc (rdtsc ()),
